api/v1/controllers/RegistrationController.js

```diff
@@ -6,7 +6,7 @@ const services = require('../services');
 const middleware = require('../middleware');
 const requests = require('../requests');
 const roles = require('../utils/roles');
-const mail = require('../utils/mail');
+const config = require('../../config');
```

Comment:
This should use the ctx module

Reply:
Fixed in e47e720dae8f4da29d900cff92733465a041dfec